### PR TITLE
Optimization: Decrease performance cost of Inferno Cannon fire particles by 36%

### PR DESCRIPTION
**Pull request by xezon**
This change decreases the cost of the Inferno Cannon fire particles by 36%.

## Original Napalm

https://github.com/TheSuperHackers/GeneralsGamePatch/assets/4720891/17dd7eca-a2c6-45ad-b7c8-305d86bbc7fd

## Patched Napalm

https://github.com/TheSuperHackers/GeneralsGamePatch/assets/4720891/2ddef2d3-863a-4f6c-9783-ece97fe1b646

## Original Black Napalm

https://github.com/TheSuperHackers/GeneralsGamePatch/assets/4720891/73154993-16b3-4f18-9c79-52045f54c903

## Patched Black Napalm

https://github.com/TheSuperHackers/GeneralsGamePatch/assets/4720891/f6c11e5d-3d8a-4d20-9218-c5f790c03e56

### CODE REVIEW

File: Patch104pZH/GameFilesEdited/Data/INI/FXList.ini [hunk 1]
```diff
@@ -882,9 +882,10 @@ FXList WeaponFX_InfernoTankShellDetonation
...
     UseCallersRadius = Yes ; Override the radius with the damage radius of the weapon.
   End
-  ParticleSystem
-    Name = BuggyMissileExplosionSmoke
-  End
+  ; Patch104p @tweak xezon 25/06/2023 Remove obsolete smoke from center of impact.
+  ;ParticleSystem
+  ;  Name = BuggyMissileExplosionSmoke
+  ;End
 End
```
Comment (commy2) [inline Patch104pZH/GameFilesEdited/Data/INI/FXList.ini:889]: Consider that some effects may still be used if one reaches the particle limit depending on settings.

Comment (xezon) [reply Patch104pZH/GameFilesEdited/Data/INI/FXList.ini:889]: I do not expect this to be a concern.

Comment (commy2) [reply Patch104pZH/GameFilesEdited/Data/INI/FXList.ini:889]: Uh, have you tried?

Comment (xezon) [reply Patch104pZH/GameFilesEdited/Data/INI/FXList.ini:889]: I would not know what to try. Looks all correct to me.

Comment (commy2) [reply Patch104pZH/GameFilesEdited/Data/INI/FXList.ini:889]: You know how when a lot of particles are drawn, some are skipped? The worry is that no particles are shown anymore when before the buggy missile explosion would've been drawn anyway.

Comment (xezon) [reply Patch104pZH/GameFilesEdited/Data/INI/FXList.ini:889]: It will still show the BuggyMissileExplosion, which is defined above.

**Comment by xezon** [comment]
Documentation added. Looks good.

**Pull request merged**